src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int maxValue = 100;

```

Comment:
if you want to create a constant it will look like this
```suggestion
    private static final int MAX_VALUE = 100;

```